baselines/fedwav2vec2/fedwav2vec2/conf/base.yaml

```diff
@@ -6,20 +6,25 @@
 pre_train_model_path: null
 # Path for label encoder file if want to ensure the same encode for every client
 label_path: docs/label_encoder.txt
+
+huggingface_model_save_path: docs/pretrained_wav2vec2
 sb_config: fedwav2vec2/conf/sb_config/w2v2.yaml # config with SpeechBrain recipe for Wav2Vec 2.0
 data_path: data # if you change this, ensure you `git cloned` the author's own repo to a directory with the new name
-rounds: 30 # global FL rounds
-local_epochs: 5 # local epochs for each client
+rounds: 100 # global FL rounds
+local_epochs: 15 # local epochs for each client
 total_clients: 1943
 server_cid: 19999
 
 # Device setup
-device: cpu
+device: cuda
+
+
 parallel_backend: false # If using multi-gpus per client (disable if using CPU)
 
 dataset:
   download_filename: TEDLIUM_release-3.tgz
-  extract_subdirectory: audio
+
+  extract_subdirectory: audio/TEDLIUM_release-3
```

Comment:
```suggestion
  extract_subdirectory: audio
```

Reply:
then, in `_csv_path_audio` in `dataset_preparation.py` replace:

```
_csv_path_audio(cfg.data_path, EXTRACT_PATH")
```
with

```
_csv_path_audio(cfg.data_path, f"{EXTRACT_PATH}/TEDLIUM_release-3")
```

It won't work otherwise.